fdbclient/NativeAPI.actor.cpp

```diff
@@ -7952,14 +7979,15 @@ Reference<TransactionLogInfo> Transaction::createTrLogInfoProbabilistically(cons
 	return Reference<TransactionLogInfo>();
 }
 
+// TODO - ljoswiak, mpilman. Discuss 2 functions below and implications on txID.
 void Transaction::setTransactionID(uint64_t id) {
```

Comment:
This needs to get updated to be a full 16 bytes. Then the places that call this need to be updated as well (one place is from the special key space).